online_competitions/settings.py

```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = '^(txxj1p(@hfc)@u7u#d1h$i##m+g*=m!wzu^%zs3r+lw4mlh_'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
toto tu malo byt tiez? ak hej tak mozes mergnut

Reply:
JJ maloaspon docasne ... potom pridam dvoje settings pre produkciu a test a spravim to ako s MP